test/coverlet.core.tests/Instrumentation/InstrumenterTests.cs

```diff
@@ -23,14 +23,10 @@ public class InstrumenterTests
         private readonly InstrumentationHelper _instrumentationHelper = new InstrumentationHelper(new ProcessExitHandler(), new RetryHelper(), new FileSystem());
         private readonly Mock<ILogger> _mockLogger = new Mock<ILogger>();
 
-        [Fact(Skip = "To be used only validating System.Private.CoreLib instrumentation")]
+        [Fact]
```

Comment:
@ViktorHofer added `System.Private.CoreLib.dll` tests on CI to validate interlocked class(was manual).